### PR TITLE
Implement CredentialCenter methods of credential management APIs

### PR DESCRIPTION
**Pull request by baixinsui**
@iskey @swaroopar Please review this PR.

### CODE REVIEW

File: modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java [hunk 1]
```diff
@@ -201,11 +250,89 @@ public CredentialDefinition getCredential(Csp csp, String userName) {
...
+
+    private CredentialDefinition fillCredentialWithDbEntity(CreateCredential createCredential,
+                                                            DeployServiceEntity deployService) {
+        createCredential.setCsp(deployService.getCsp());
+        createCredential.setUserName(deployService.getUserName());
+        return transCredential(createCredential);
+
+    }
+
+    private CredentialDefinition transCredential(CreateCredential createCredential) {
```
Comment (swaroopar) [inline modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java:324]: createCredentialDefinitionObject is a better name here

File: modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCacheManager.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+     * @return true or false.
+     */
+    public boolean isEmpty() {
+        return CACHES.isEmpty();
+    }
+
+    /**
+     * Put cache.
+     *
+     * @param key   key
```
Comment (swaroopar) [inline modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCacheManager.java:46]: in this entire class.. we must replace "key" with a more concrete value.
One idea would be.. to have a new object which contains - Csp and username. and use that in all method definitions. 
and move the getCacheKey method inside this class.

**Review by swaroopar — CHANGES_REQUESTED** [review]
comments added

File: modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java (hunk 1 above)
Comment (baixinsui) [reply modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java]: Ok.

File: modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCacheManager.java (hunk 2 above)
Comment (baixinsui) [reply modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCacheManager.java:46]: Done.

**Review by swaroopar — APPROVED** [review]
looks good.

**Review by iskey — APPROVED** [review]
looks good.

**Pull request merged**